app/controllers/registrations_controller.rb

```diff
@@ -210,16 +210,25 @@ def do_add
     end
     ActiveRecord::Base.transaction do
       user, locked_account_created = user_for_registration!(params[:registration_data])
-      registration = @competition.registrations.find_or_initialize_by(user_id: user.id)
-      raise I18n.t("registrations.add.errors.already_registered") unless registration.new_record?
-      registration_comment = params.dig(:registration_data, :comments)
-      registration.assign_attributes(comments: registration_comment) if registration_comment.present?
-      registration.assign_attributes(accepted_at: Time.now, accepted_by: current_user.id)
-      params[:registration_data][:event_ids]&.each do |event_id|
-        competition_event = @competition.competition_events.find { |ce| ce.event_id == event_id }
-        registration.registration_competition_events.build(competition_event_id: competition_event.id)
+      if @competition.uses_new_registration_service?
+        Microservices::Registrations.add_registration(@competition.id,
+                                                      user.id,
+                                                      params[:registration_data][:event_ids],
+                                                      params[:registration_data][:comments],
+                                                      "accepted",
+                                                      @current_user.id)
+      else
```

Comment:
I don't have the "Hide whitespace" feature on my phone, so GitHub makes it seem like all these lines changed. I suspect that the only actual change is indentation because of the if-else, can you please confirm?

Reply:
yes. I just enabled white whitespace and all those changes dissappeared

Reply:
![image](https://github.com/user-attachments/assets/66374b6f-fe74-4fcc-935e-1609de323ab2)